### PR TITLE
Feature/parse url params

### PR DESCRIPTION
**Pull request by gamalielhere**
### Feature

- [x] Updated CHANGELOG.md
- [x] Add PR label

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ router.beforeResolve((to, ___, next) => {
   ) {
     next();
   } else {
+    if (Object.keys(to.query).length > 1) {
+      store.dispatch('saveQueryVal', to.query);
```
Comment (kvhnuke) [inline src/router/index.js]: we have to validate the queries against xss injections

Comment (kvhnuke) [reply src/router/index.js]: https://www.npmjs.com/package/xss

File: src/layouts/InterfaceLayout/containers/SendOfflineContainer/SendOfflineContainer.vue [hunk 2]
```diff
@@ -326,6 +327,26 @@ export default {
       this.createDataHex(null, null, newVal);
     }
   },
+  mounted() {
+    if (Object.keys(this.linkQuery).length > 0) {
+      const { data, to, value, gaslimit, gas, tokensymbol } = this.linkQuery;
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/SendOfflineContainer/SendOfflineContainer.vue]: did you get a hance to check what will happen if some of the values are undefined, I think we should check whether they are undefined and set to default values

Comment (gamalielhere) [reply src/layouts/InterfaceLayout/containers/SendOfflineContainer/SendOfflineContainer.vue]: Yeah. I tried it with just a data field and it changed just that iirc. But I'll try it again

Comment (gamalielhere) [reply src/layouts/InterfaceLayout/containers/SendOfflineContainer/SendOfflineContainer.vue]: @kvhnuke new push checks values now.

File: src/layouts/InterfaceLayout/containers/SendCurrencyContainer/SendCurrencyContainer.vue [hunk 3]
```diff
@@ -350,6 +351,26 @@ export default {
...
+    if (Object.keys(this.linkQuery).length > 0) {
+      const { data, to, value, gaslimit, tokensymbol } = this.linkQuery;
+      const foundToken = this.tokensWithBalance.find(item => {
+        return item.symbol.toLowerCase() === tokensymbol.toLowerCase();
+      });
+      this.value = value ? new BigNumber(value).toFixed() : 0;
+      this.data = data ? data : '0x';
+      this.hexAddress = to ? to : '';
+      this.address = to ? to : '';
+      this.gasLimit = gaslimit ? new BigNumber(gaslimit).toString() : '2100';
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/SendCurrencyContainer/SendCurrencyContainer.vue]: 21000

File: src/layouts/InterfaceLayout/containers/SendOfflineContainer/SendOfflineContainer.vue [hunk 4]
```diff
@@ -326,6 +327,26 @@ export default {
...
+      const foundToken = this.tokens.find(item => {
+        return item.symbol.toLowerCase() === tokensymbol.toLowerCase();
+      });
+      this.toAmt = value ? new BigNumber(value).toFixed() : 0;
+      this.toData = data ? data : '0x';
+      this.address = to ? to : '';
+      this.gasLimit = gaslimit ? new BigNumber(gaslimit).toString() : '2100';
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/SendOfflineContainer/SendOfflineContainer.vue]: 21000

**Pull request merged**